### ISSUE TITLE
guided entity creator

### ISSUE THREAD

**Issue by XenotropicDev** [XenotropicDev/TheOracle2#31]
The general idea would be that for things like planets or NPCs the user could say I generically want something with x features but the rest should be random.

Not sure if this is really needed once the all the other planned features are complete. Also not sure if it should just use the integrated Oracles, or a template class that would handle the options.

**Comment by rsek** [comment]
> Not sure if this is really needed once the all the other planned features are complete. Also not sure if it should just use the integrated Oracles, or a template class that would handle the options.

yeah, i'm leaning that way myself. if we can provide the right oracle roll options to add from a dropdown or something, it could be feel a little more organic, like tabletop planet/npc generation (particularly in light of the "peeling the onion" philosophy of the game), rather than having to decide from the outset that something is a "formal" planet/npc.

maybe it'd be feasible way to select specific oracle results via ephemerals? like, select an oracle, get a prompt with one or more dropdowns enabling the selection of individual rows?

any structure on top of that could then be optional, like tacking a Connection track on to something that already contains a bunch of oracle rolls, or that simply alters the embed display by putting the name in the embed's title Field, etc

**Comment by rsek** [comment]
i'm currently working on an implementation w/ the new dataforged oracle format, with an eye towards making the sort of stuff i describe above easier. the oracle shortcuts offered in the select menu will be chosen programatically (taking cues from the Suggestion key of dataforged, the categories of existing rolls, etc) rather than anything hand-coded.

my plan is to limit it to grouping oracles without any fancy "convert to starship" things or "select from table" for the initial offering, mainly so this PR doesn't grow totally out of control.

some means of selecting table entries instead of rolling (probably "chained" ephemerals) is something i could build on top of that.
